velox/docs/develop/TpchBenchmark.rst

```diff
@@ -115,7 +115,6 @@ Top Optimization Recommendations
    "num_splits_per_file", "Row Group Size of Data", "Row Group Size of Data"
    "max_coalesce_bytes", "Minimum of 90MB", "Minimum of 90MB"
    "max_coalesce_distance_bytes", "Workload dependent**", "Workload dependent**"
-   "parquet_prefetch_rowgroups", "Best is 1", "Best is 1"
```

Comment:
Let's keep this and add a flag `parquet_prefetch_rowgroups` to TpchBenchmark.cpp.
We can then pass this via the connector properties just like `max_coalesced_bytes`
Also please fix typo here `max_coalesce_bytes` -> `max-coalesced-bytes`
`max_coalesce_distance_bytes` -> `max_coalesced_distance_bytes`